README.md

```diff
@@ -185,6 +172,19 @@ spec:
       storage: 1Mi
 ```
 
+# Build and publish your own container image
+
+To build your own custom container image from this repository, you will have to build and push the nfs-subdir-external-provisioner image using the following instructions.
+
+```sh
+make build
+make container
+# `nfs-subdir-external-provisioner:latest` will be created. 
+# To upload this to your custom registry, say `quay.io/myorg`, you can use
+# docker tag nfs-subdir-external-provisioner:latest quay.io/myorg/nfs-subdir-external-provisioner:latest
+# docker push quay.io/myorg/nfs-subdir-external-provisioner:latest
```

Comment:
when you are releasing `:latest`, it will be multi-arch image. if ill follow this instructions ill get multi-arch or amd64 image?

Reply:
At the moment, there is no latest. The above custom steps will create the image with the arch on which it is built.